instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpSpanNameExtractor.java

```diff
@@ -33,13 +33,10 @@ private HttpSpanNameExtractor(HttpCommonAttributesGetter<REQUEST, ?> getter) {
 
   @Override
   public String extract(REQUEST request) {
-    String route = extractRoute(request);
-    if (route != null) {
-      return route;
-    }
     String method = getter.getMethod(request);
+    String route = extractRoute(request);
     if (method != null) {
-      return "HTTP " + method;
+      return route == null ? method : method + " " + route;
     }
     return "HTTP request";
```

Comment:
wdyt of

```suggestion
    return "HTTP";
```

or

```suggestion
    return "UNKNOWN";
```

Reply:
I think I like `HTTP` more; if nothing else is provided, no attributes at all, at least it says that it was an HTTP call.